tests/test_algorithms.py

```diff
@@ -0,0 +1,86 @@
+from collections import namedtuple
+
+import pytest
+
+from pytest_split import algorithms
+
+
+item = namedtuple("item", "nodeid")
+
+
+class TestAlgorithms:
+    @pytest.mark.parametrize("algo_name", algorithms.ALGORITHMS)
+    def test__split_test(self, algo_name):
```

Comment:
Let's keep consistency, other test modules seem to use single `_` after `test` 🙂

Reply:
It is consistent in the sense that I use the format: `test_{func}_{does_something}_{when}`.
And in this case the func is called `_split_test`.

Please let me know if you still prefer the change.